backend/inspirehep/search/api.py

```diff
@@ -43,7 +51,7 @@ def get_source(self, uuid, **kwargs):
         :type uuid: UUID
         :returns: dict
         """
-        return es.get_source(index=self.Meta.index, id=uuid, **kwargs)
+        return es.get_source(index=self.alias, id=uuid, **kwargs)
```

Comment:
this means that we'll search in all records?

Reply:
Discussed IRL